code/modules/surgery/organs/liver.dm

```diff
@@ -250,4 +250,8 @@
 	if(prob(emp_vulnerability/severity)) //Chance of permanent effects
 		organ_flags |= ORGAN_SYNTHETIC_EMP //Starts organ faliure - gonna need replacing soon.
 
-
+/obj/item/organ/liver/lizard
+	name = "lizard liver"
+	icon_state = "liver"
+	desc = "Due to a low number of natural poisons on Tizira, lizard livers have a lower tolerance for poisons when compared to human ones."
```

Comment:
```suggestion
	desc = "Due to the low number of natural poisons on Tizira, lizard livers have a lower tolerance for poisons than compared to human ones."
```

Reply:
thats not how grammar works you cant replace when with than and act like the sentence works the same

Reply:
my bad, I meant to remove the "compared to" as well